File: DuDoong-Infrastructure/src/main/java/band/gosrock/infrastructure/outer/api/tossPayments/exception/PaymentsEnumNotMatchException.java
package band.gosrock.infrastructure.outer.api.tossPayments.exception;


import band.gosrock.common.exception.DuDoongCodeException;
import band.gosrock.common.exception.GlobalErrorCode;

public class PaymentsEnumNotMatchException extends DuDoongCodeException {
    public static final DuDoongCodeException EXCEPTION = new PaymentsEnumNotMatchException();

    private PaymentsEnumNotMatchException() {
        super(GlobalErrorCode.TOSS_PAYMENTS_ENUM_NOT_MATCH);
    }
}

File: DuDoong-Infrastructure/src/main/kotlin/band/gosrock/infrastructure/outer/api/tossPayments/exception/PaymentsEnumNotMatchException.kt
package band.gosrock.infrastructure.outer.api.tossPayments.exception

import band.gosrock.common.exception.DuDoongCodeException
import band.gosrock.common.exception.GlobalErrorCode

class PaymentsEnumNotMatchException private constructor() :
    DuDoongCodeException(GlobalErrorCode.TOSS_PAYMENTS_ENUM_NOT_MATCH) {
    companion object {
        @JvmField val EXCEPTION: DuDoongCodeException = PaymentsEnumNotMatchException()
    }
}
